lua/tint.lua

```diff
@@ -39,6 +41,9 @@ __.default_config = {
     focus = { "WinEnter" },
     unfocus = { "WinLeave" },
   },
+  should_create_extra_tint = function()
```

Comment:
What is an example of how you would use this function, and its arguments?

`winid` makes sense - you may want to get the `&ft` of a window and determine if you want it to be tinted differently.

I do not understand why the user would need `untint_ns_id` and `ns_suffix` to be passed to them.

Reply:
good point :) in my latest draft, i use only one param, and it is the "untint_ns_id", so the original ns_id of the current window. I think it might be more logical, since i now only call the user-method, if no tinted version of ns_id already exists. so the "decision" really depends on the ns_id, not the window.

Still, from users perspective the decision by winid (and in consequence e.g. by filetype) seems easier to understand and implement.

My current version:

```
        should_create_extra_tint = function(ns_id)
          local hl_ns_name = nil
          for ns_name, hl_ns_id in pairs(vim.api.nvim_get_namespaces()) do
            if hl_ns_id == ns_id then
              hl_ns_name = ns_name
              break
            end
          end
          if hl_ns_name ~= nil and string.find(hl_ns_name, "styler_") then
            return true
          else
            return false
          end
        end,
```

...so yeah, I guess I will refactor and send winid instead. What do you think, should the function be called always, per window, or only if no extra_namespace exists?

Reply:
arg for should_create_extra_tint is now winid, example see DOC.md or doc/tint.txt